src/platform/portduino/PortduinoGlue.cpp

```diff
@@ -94,6 +100,39 @@ void portduinoSetup()
     printf("Setting up Meshtastic on Portduino...\n");
 
 #ifdef ARCH_RASPBERRY_PI
+    YAML::Node yamlConfig;
+    try {
+        yamlConfig = YAML::LoadFile("config.yaml");
```

Comment:
This is not a good (but inefficient and expensive) way in C++ to check if a file exist. Pls. don't misuse the YAML class for this and run purposely into exceptions. Also, there are several types of YAML exceptions, e.g. it could be that there is a syntax error in the file. 
It's better (and the code structure becomes easier) to use the standard C++ way to find the file and if it is found then use YAML to parse the file.

Reply:
That's fair. Something like access() a better option?